src/document-creator/document-creator.mdx

```diff
@@ -83,9 +83,30 @@ As of now, we only cater to 3 networks.
 
 ## Wallet field
 
-The `"wallet"` field is a string that refers to your ethereum wallet.
+The `"wallet"` field is a string that refers to your ethereum wallet. 
 
-If you already have an existing wallet.json, the only thing you need to do is to run JSON.stringify over the contents in order to serialise it into a string.
+If you don't have a wallet, you can refer to [Open Attestation's documentation](https://github.com/Open-Attestation/open-attestation-cli#wallet) to create a wallet.
+_Note: you will need to have Open Attestation Cli installed._ This will give you a wallet.json.
```

Comment:
```suggestion
_Note: you will need to have Open Attestation Cli installed._ After running OA CLI `wallet create` command, this will give you a wallet.json.
```